src/lib/esnext.iterator.d.ts

```diff
@@ -0,0 +1,133 @@
+interface NativeIterator<T, TReturn = void, TNext = undefined> extends Iterator<T, TReturn, TNext> {

```

Comment:
The `export {}` trick mentioned in my earlier comment can address the `abstract next()` method definition.